contrib/mkimage-arch.sh

```diff
@@ -14,20 +17,31 @@ hash expect &>/dev/null || {
 	exit 1
 }
 
-
 export LANG="C.UTF-8"
 
 ROOTFS=$(mktemp -d ${TMPDIR:-/var/tmp}/rootfs-archlinux-XXXXXXXXXX)
 chmod 755 $ROOTFS
 
+# required packages
+PKGREQUIRED=(
+    bash
+    haveged
+    pacman
+    pacman-mirrorlist
+)
+
 # packages to ignore for space savings
 PKGIGNORE=(
-    cryptsetup
-    device-mapper
     dhcpcd
+    diffutils
+    file
+    inetutils
```

Comment:
Hm.. looks like this whole block of the script was already using spaces instead of tabs; perhaps you can fix all of those?

Reply:
updated